### PR TITLE
Use region defaulting wherever possible

### PR DESCRIPTION
**Pull request by fiunchinho**
### Checklist

- [X] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/test create
/test upgrade

### CODE REVIEW

**Comment by marians** [comment]
If the default value is needed, then we should also remove it from values.schema.json and, as a consequence, from values.yaml.

File: helm/cluster-aws/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: cluster-aws
 description: A helm chart for creating Cluster API clusters with the AWS infrastructure provider (CAPA).
 icon: https://s.giantswarm.io/app-icons/aws/1/dark.svg
 type: application
-version: [[ .Version ]]
+version: 0.1.0
```
Comment (AndiDog) [inline helm/cluster-aws/Chart.yaml:6]: This change shouldn't be committed

Comment (fiunchinho) [reply helm/cluster-aws/Chart.yaml:6]: Ooops! Good catch.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Added property schema for subnetTags objects
   - Added default values
 
+### Fixed
+
+- Use region defaulting wherever possible.
```
Comment (marians) [inline CHANGELOG.md:21]:
```suggestion
- Use region defaulting wherever possible, removing `region` from schema.
```

**Review by marians — APPROVED** [review]

**Pull request merged**